### PR TITLE
[16.0][ADD] l10n_fr_department_product_origin

### PR DESCRIPTION
**Pull request by legalsylvain**
based on the pending PR : https://github.com/OCA/product-attribute/pull/1389
(+ changes done in https://github.com/acsone/product-attribute/pull/8)

### Description

This module extends the functionality of Product Origin module to add french department on products.

product_form

CC : @quentinDupont, @alexis-via, @acsonefho

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+git+https://github.com/OCA/product-attribute@refs/pull/1389/head#subdirectory=setup/product_origin
```
Comment (legalsylvain) [inline test-requirements.txt]: Hi @sbidoul. 

Question : once setup folder has been removed, replaced by toml file, what is the correct syntax to write such dependency in the futur ? 

thanks !

Comment (sbidoul) [reply test-requirements.txt]: @legalsylvain `subdirectory` is the directory where the python project is. The python project is where the setup.py or pyproject.toml is. So, just replace `setup/product-origin` with `product_origin`.

See also https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

Comment (legalsylvain) [reply test-requirements.txt]: Thanks ! very clear.

**Pull request merged**